### PR TITLE
Add tutorial for www puhti

### PR DESCRIPTION
**Pull request by yetulaxman**

### CODE REVIEW

**Comment by attesillanpaa** [comment]
Can you add a whats-new entry for this? I think this is a really cool tutorial. If this is your first whats-new, please read how it's done (there's a trick).

File: docs/computing/webinterface/jupyter-for-courses.md [hunk 1]
```diff
@@ -50,3 +50,6 @@ partition: "interactive"
 local_disk: 32
 mem: "16GB"
 ```
+### Tutorials
+[Tutorial example for course trainers](https://github.com/CSCfi/Jupyter_www_puhti): This tutorial is useful for course trainer
```
Comment (robinkar) [inline docs/computing/webinterface/jupyter-for-courses.md:54]:
```suggestion
[Tutorial example for course organizers](https://github.com/CSCfi/Jupyter_www_puhti): This tutorial is useful for course organizers
```
Organizer sounds a bit better to me than trainer

File: docs/computing/webinterface/jupyter-for-courses.md [hunk 2]
```diff
@@ -50,3 +50,6 @@ partition: "interactive"
 local_disk: 32
 mem: "16GB"
 ```
+### Tutorials
+[Tutorial example for course organizers](https://github.com/CSCfi/Jupyter_www_puhti): This tutorial is useful for course organizers
+who wants to provide custom Jupyter notebooks *via* Puhti web interface.
```
Comment (robinkar) [inline docs/computing/webinterface/jupyter-for-courses.md:55]:
```suggestion
who want to provide custom Jupyter notebooks *via* Puhti web interface. 
```
Small fix

**Review by attesillanpaa — APPROVED** [review]
nice!

**Pull request merged**